scripts/run.py

```diff
@@ -1814,7 +1814,7 @@ def _report_xunit_result(self, testsuite_name, testcase_name, testcase_runtime,
                     failure_message += "\n\n"
                     failure_message += "#" * 83 + "\n"
                     failure_message += "########### Problems encountered extracting Java messages. " \
-                                       "Please alert the QA team.\n"
+                                       "Massive Jenkins or test failure.\n"
```

Comment:
Some tests failed spectacularly and there is no Java output associated with that.  Probably the whole cluster shut down before the test finished.  Hence, the warning message.  However, previously Brandon wants to alert the QA team but I don't think anyone can do anything about it.  So, instead of the misleading message, I change the error message printout.  No big deal.